docs/stats.md

```diff
@@ -0,0 +1,30 @@
+## accessing stats
+
+stats can be accessed via 
+1. `pumactl`
+2. Puma.stats when in non-worker mode
+3. Puma.stats when in worker mode (should not be invoked from worker process, right?)
```

Comment:
I think officially these modes are called "single mode" and "cluster mode". The ways you can access stats are the same, they just have different formats depending on the mode.

Reply:
yup let's keep the terminology consistent w/single+cluster.